style.css

```diff
@@ -0,0 +1,113 @@
+@media (min-width: 370px) {
+  * {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+
+  body {
+    background-color: #f7f7f9;
```

Comment:
- [ ] As stated by the previous reviewer, kindly change the background color. ` background-color: #fff;`